go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/improbable-eng/grpc-web v0.12.0
 	github.com/logrusorgru/aurora v0.0.0-20200102142835-e9ef32dff381
-	github.com/onflow/cadence v0.10.0
+	github.com/onflow/cadence v0.10.1
 	// this references: https://github.com/onflow/flow-go/tree/feature/multiple-contract-support.
-	github.com/onflow/flow-go v0.10.2-0.20201026185922-70cd25209148
-	github.com/onflow/flow-go-sdk v0.12.0
+	github.com/onflow/flow-go v0.11.1-0.20201109210205-312549711f62
```

Comment:
I think we should tag `onflow/flow-go` with a semantic version to avoid issues with missing commits in the future

Reply:
There is, `feature/multiple-contract-support`, as the comment mentions. Unfortunately, Go doesn't support pinning against branches or custom tag names. Once that feature PR is merged, we will switch back to flow-go's last released version.

Reply:
Oh I see, my bad. Makes sense!